airbyte-integrations/connectors/source-google-workspace-admin-reports/source_google_workspace_admin_reports/api.py

```diff
@@ -123,28 +160,39 @@ def get_params(self) -> Dict:
         return params
 
     def process_response(self, response: Dict) -> Iterator[dict]:
-        return response.get("items", [])
+        activities = response.get("items", [])
+        for activity in activities:
+            activity_id = activity.get("id", {})
+            if "time" in activity_id:
+                # place time property in top level
+                activity["time"] = activity_id["time"]
+            yield activity
 
     def list(self, fields: Sequence[str] = None) -> Iterator[dict]:
         params = self.get_params()
```

Comment:
Shouldn't this include the state in the request?

Reply:
It includes state in params. start_time is sent in API request. And start_time is set to state value
https://github.com/airbytehq/airbyte/pull/2744/files#diff-03cf480d7c7d443c6f5c02a205e4a694a5745824bd494b9d514cf7315c56ff8eR135

Reply:
ok I see now, thanks for the clarification!